Makefile

```diff
@@ -80,3 +82,11 @@ test-ci: ## Run the unit tests (used in the CI)
 .PHONY: test-e2e
 test-e2e: ## Run the integration tests (used in the CI)
 	./test_e2e.sh
+
+.PHONY: test-coverage
+test-coverage:  ## Run coveralls
+	- rm -rf *.out
+	echo "mode: count" > coverage-all.out
+	$(foreach pkg,$(PKGS_COVER),\
+		go test -failfast -tags=integration -coverprofile=coverage.out -covermode=count $(pkg) || exit 1;\
+		tail -n +2 coverage.out >> coverage-all.out;)
```

Comment:
where does this come from?  The goveralls docs don't seem to think this is necessary.  It at least needs a comment.

Reply:
I simplified and added the comments. Please, let me know what do you think now.

Reply:
actually, I think we can simplify this further: `go test -failfast -tags=integration -coverprofile=coverages.out -covermode=count ./pkg/... ./cmd/...`

Reply:
It is true 👍 Getter better how it works too :-)